<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>真值系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="bt_videodis">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>视频显示</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/camera.png</normaloff>:/image/camera.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_lidardis">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>激光雷达</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/lidar.png</normaloff>:/image/lidar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_radardis">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>毫米波雷达</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/radar.png</normaloff>:/image/radar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_dev_replay">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>视频回放</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/replay.png</normaloff>:/image/replay.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_dev_calibration">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>设备标定</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/calibration.png</normaloff>:/image/calibration.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_sensor_fusion">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>融合显示</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/fusion.png</normaloff>:/image/fusion.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_match_fusion">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #ffffff;
	#boder:none;
}
QPushButton:hover{
	background-color: #ecf5ff;
	color:#409eff;
}
QPushButton:pressed,QPushButton:checked{
	border:0px;
	color:#409eff;
}
</string>
        </property>
        <property name="text">
         <string>实时配准</string>
        </property>
        <property name="icon">
         <iconset resource="../source.qrc">
          <normaloff>:/image/merge.png</normaloff>:/image/merge.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>13</number>
        </property>
        <property name="leftMargin">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QPushButton" name="bt_cameraparam">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;弃用&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:0px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../source.qrc">
            <normaloff>:/image/videodisplay.png</normaloff>:/image/videodisplay.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_eventSet">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;事件记录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:0px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../source.qrc">
            <normaloff>:/image/check.png</normaloff>:/image/check.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_systemparam">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border:0px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../source.qrc">
            <normaloff>:/image/system.png</normaloff>:/image/system.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stack_sidewidget">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stack_mainwidget">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>600</height>
         </size>
        </property>
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page_4"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="../source.qrc"/>
 </resources>
 <connections/>
</ui>
